src/test/scala/is/hail/expr/ir/ExtractAggregatorsSuite.scala

```diff
@@ -52,13 +52,17 @@ class ExtractAggregatorsSuite {
     val tAgg = TAggregable(hailType[IN], Map("scope0" -> (0, hailType[SCOPE0])))
 
     // special arguments: region, aggregator, element, element missingness
-    val (post, aggResultStruct, aggregators) = ExtractAggregators(ir, tAgg)
+    val (post, aggResultStruct, aggIR, rvAggs) = ExtractAggregators(ir, tAgg)
 
-    val seqOps = aggregators.map { case (ir, agg) =>
-      val fb = EmitFunctionBuilder[Region, RegionValueAggregator, IN, Boolean, SCOPE0, Boolean, Unit]
-      Emit(ir, fb, 2, tAgg)
+    val seqOps = {
+      val fb = EmitFunctionBuilder[Region, Array[RegionValueAggregator], IN, Boolean, SCOPE0, Boolean, Unit]
 
-      (agg, fb.result(Some(new java.io.PrintWriter(System.out)))())
+      val s = Subst(aggIR, Env.empty[IR].bind("scope0", In(1, hailType[SCOPE0])))
+      Emit(s, fb, 2, tAgg)
+
+      fb.result(
+        // Some(new java.io.PrintWriter(System.out))
```

Comment:
We can change this to `None` if we don't want the output.

Reply:
I find I often want to comment this in so I thought I'd leave it.  Can delete if you prefer.